egs/wsj/s5/steps/rnnlmrescore.sh

```diff
@@ -66,7 +66,7 @@ elif [ ! -f $oldlm ]; then
     exit 1;
 fi
 
-for f in $rnndir/rnnlm $data/feats.scp $indir/lat.1.gz; do
+for f in $data/feats.scp $indir/lat.1.gz; do
```

Comment:
I don't like the fact that this script is being used for so many different tasks.  I think it would be better to just copy and modify this script, putting it in the rnnlm/ script directory.   That would make it possible to change the way the I/O works to be more Kaldi-like, i.e. you wouldn't have to remove the first field (the key) from those files.

Reply:
Hmm I thought about that but eventually didn't because this file is quite large. What I did is at stage 6, adding a new script called rnnlm/rnnlm_compute_scores.sh , which is called when kaldi-rnnlm is used. 
Do you want me to copy a large part of this file (all the lattice operations) to scripts/rnnlm/ ?